rules/helm/package.tmpl.rb

```diff
@@ -0,0 +1,78 @@
+require "fileutils"
+require "open3"
+require "tmpdir"
+
+# Variables expanded by Bazel.
+package_dir = "[[package_dir]]"
+multipath_sep = "[[multipath_sep]]"
+tarsStr = "[[tars]]"
+generatedStr = "[[generated]]"
+helm = "[[helm]]"
+output_tgz = "[[output_tgz]]"
+
+# Variables expanded by gomplate.
+git_commit_short = '{{ (ds "workspace_status").STABLE_GIT_COMMIT_SHORT }}'
+git_branch = '{{ (ds "workspace_status").STABLE_GIT_BRANCH }}'
+
+# Create the temporary build directory for joining all the pieces required for packaging.
+tmp_build_dir = Dir.mktmpdir("build-", Dir.getwd)
+begin
+  build_dir = File.join(tmp_build_dir, package_dir)
+  FileUtils.mkdir_p(build_dir)
+
+  # Copy the source static files to the temporary build directory.
+  Dir.glob(File.join(package_dir, "**", "*")) do |file|
+    dest = File.join(tmp_build_dir, file)
+    file = File.readlink(file) while File.ftype(file) == "link"
+    if File.ftype(file) == "directory"
+      FileUtils.mkdir_p(dest)
+    else
+      FileUtils.cp(file, dest)
+    end
+  end
+
+  # Extract the tar files into the temporary build directory.
+  tars = tarsStr.split(multipath_sep)
+  tars.each do |file|
+    file = File.readlink(file) while File.ftype(file) == "link"
+    pid = Process.spawn("tar xf '#{file}' -C '#{build_dir}'")
+    _, status = Process.wait2 pid
+    exit 1 unless status.success?
+  end
+
+  # Copy the generated files into the temporary build directory.
+  generated = generatedStr.split(multipath_sep)
+  generated.each do |file|
+    file = File.readlink(file) while File.ftype(file) == "link"
+    FileUtils.cp(file, build_dir)
+  end
+
+  pid = Process.spawn("'#{helm}' init --client-only")
+  _, status = Process.wait2 pid
+  exit 1 unless status.success?
+
+  # Handle chart versioning based on git state.
+  version = "v0.0.0-#{git_commit_short}"
```

Comment:
Thinking about it some more, fallback should probably be closer to `git describe` output (so that the _current_ builds would look something like 0.2.0-derived).

But I'm totally happy to punt that to a future PR.